tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -850,4 +852,81 @@ public void submit() {
     }
   }
 
+  /**
+   * Object used to keep some data when a configuration change happens and the activity is
+   * re-created.
+   * It's boiler-plate but this is how to save View state.
+   */
+  private static class SavedState extends BaseSavedState {
+    /* This will store the current scale and position */
+    float scale;
```

Comment:
can we use the `m` prefix on member variables?  `mScale`, `mSavedCenterX`, `mSavedCenterY`